frontend/cypress/integration/metadata-spec.js

```diff
@@ -0,0 +1,176 @@
+
+		cy.get('[data-qa="pnx-dataset-form"] > div.row > div.col-md-8 > div > div.card-body > div:nth-child(10) > pnx-nomenclature > ng-select > div > div > div.ng-input').click()
+		cy.get('[data-qa="Autre"]').click()
+		cy.get("[data-qa='pnx-dataset-form'] > div.mr-1.mt-1 > button.mat-focus-indicator.button-success.ml-1.mat-raised-button.mat-button-base.mat-button-disabled").should('be.disabled')
+	
+		cy.get('[data-qa="pnx-dataset-form"] > div.row > div.col-md-8 > div > div.card-body > div:nth-child(11) > div:nth-child(1) > pnx-nomenclature > ng-select > div > div > div.ng-input').click()
+		cy.get('[data-qa="Ne sait pas : L\'information indiquant si la Donnée Source est publique ou privée n\'est pas connue."]').click()
+		cy.get("[data-qa='pnx-dataset-form'] > div.mr-1.mt-1 > button.mat-focus-indicator.button-success.ml-1.mat-raised-button.mat-button-base.mat-button-disabled").should('be.disabled')
+		
+		cy.get('[data-qa="pnx-dataset-form"] > div.row > div.col-md-8 > div > div.card-body > div:nth-child(11) > div:nth-child(2) > pnx-nomenclature > ng-select > div').click()
+		cy.get('[data-qa="Jeu de données"]').click()
+
+		cy.get('[data-qa="pnx-dataset-form"] > div.mr-1.mt-1 > button.mat-focus-indicator.button-success.ml-1.mat-raised-button.mat-button-base').click()
+	})
+
+	it.only('should new "jeux de données" created', () => {
+		cy.get('[data-qa="pnx-metadata-dataset-name"]').contains(newJdd.name)
+		cy.get('[data-qa="pnx-datasets-card"] > div:nth-child(1) > div > button').click()
+	})
+
+	it('should delete the new "jeux de données"', async () => {
+    cy.get('[data-qa="pnx-metadata"] > div > div.card-body > div:nth-child(2) > div.form-group.col-sm-7 > input').clear()
+		cy.get('[data-qa="pnx-metadata-acq-framework-header-1"]').click()
+		const myCadreAcq = await promisify(cy.get('[data-qa="pnx-metadata-acq-framework-header-1"]'))
+    const id = myCadreAcq[0].parentElement.childNodes[1].firstChild.firstChild.childNodes[1].childNodes[0].childNodes[0].innerText
+		cy.get('[data-qa="pnx-metadata-jdd-nb-obs-'+id+'"]').contains("0")
+		cy.get('[data-qa="pnx-metadata-jdd-delete-'+id+'"]').click()
+		cy.get("#mat-dialog-0 > gn-confirmation-dialog > div.mat-dialog-actions > button.mat-focus-indicator.mat-raised-button.mat-button-base.mat-warn.cdk-focused.cdk-program-focused").click()
+	})
+
+	it('should display data of the "cadre d\'acquisition"', async () => {
+    cy.get('[data-qa="pnx-metadata-acq-framework-header-0"]').click()
+		cy.get('[data-qa="pnx-metadata-jdd-display-data-1"]').click()
+		const listData = await promisify(cy.get("[data-qa='pnx-synthese'] > div > div.col-sm-12.col-md-5.padding-sm > pnx-synthese-list > ngx-datatable > div > datatable-body > datatable-selection > datatable-scroller"))
+		const elements = Array.from(listData[0].children);
+		elements.forEach(data => {
+			expect(data.children[0].children[1].children[4].innerText).contains("Contact aléatoire tous règnes confondus")
+		})
+	})
+
+})
```

Comment:
L'ordre des tests ne semble pas respecté.
Le "should create a new "jeux de données" est lancé en 1er ,du coup rien ne marche car il dépend du "should create a new "cardre d\'acquisition"  (qui est d’ailleurs commenté ? - mais en le décommentant, même problème)

Reply:
tous ceux où il y a un only ont en effet été corrigé, pb résolu chez nous